huaweicloud/resource_huaweicloud_networking_secgroup_v2_test.go

```diff
@@ -155,6 +176,16 @@ resource "huaweicloud_networking_secgroup_v2" "secgroup_1" {
 }
 `
 
+func testAccNetworkingV2SecGroup_epsId() string {
+	return fmt.Sprintf(`
+	resource "huaweicloud_networking_secgroup_v2" "secgroup_1" {
```

Comment:
better to remove the resource version v2

Reply:
fixed